public/components/data_connections/components/access_control_tab.tsx

```diff
@@ -0,0 +1,205 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiText,
+  EuiHorizontalRule,
+  EuiBottomBar,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import React, { useEffect, useState } from 'react';
+import { EuiPanel } from '@elastic/eui';
+import {
+  ACCELERATION_ALL,
+  ACCELERATION_RESTRICT,
+  QUERY_ALL,
+  QUERY_RESTRICT,
+} from 'common/constants/data_connections';
+import { AccessControlCallout } from './access_control_callout';
+import { coreRefs } from '../../../../public/framework/core_refs';
+import { QueryPermissionsFlexItem } from './query_permissions_flex_item';
+import { AccelerationPermissionsFlexItem } from './acceleration_permissions_flex_item';
+
+export const AccessControlTab = () => {
+  const [mode, setMode] = useState<'view' | 'edit'>('view');
+  const [roles, setRoles] = useState<Array<{ label: string }>>([]);
+  const [selectedQueryPermissionRoles, setSelectedQueryPermissionRoles] = useState<
+    Array<{ label: string }>
+  >([]);
+  const [selectedAccelerationPermissionRoles, setSelectedAccelerationPermissionRoles] = useState<
+    Array<{ label: string }>
+  >([]);
+  const [queryPermissionRadioSelected, setQueryRadioIdSelected] = useState(`query-1`);
+  const [accelerationPermissionRadioSelected, setAccelerationRadioIdSelected] = useState(
+    `acceleration-1`
+  );
+
+  const [finalQueryPermissionsPlaceHolder, setFinalQueryPermissionsPlaceHolder] = useState<
+    Array<{ label: string }>
+  >([]);
+  const [
+    finalAccelerationPermissionsPlaceHolder,
+    setFinalAccelerationPermissionsPlaceHolder,
+  ] = useState<Array<{ label: string }>>([]);
```

Comment:
Placeholder to update the state (this will be replaced with API call to POST, and then showing the list will be the result of the API call to GET)